fastlane/lanes/release.rb

```diff
@@ -163,7 +167,11 @@
     configure_apply(force: is_ci)
 
     check_translation_progress_all unless is_ci
-    download_translations
+    download_translations(
+      # skip lint in CI as a workaround to the task running in an agent that fails to download the Gradle version this project needs.
+      # The long term fix is to upgrade Gradle here.
+      run_lint: is_ci == false
+    )
```

Comment:
I intentionally duplicated the comment and kept the decision logic outside `download_translations` as a little attempt to keep the issue visible.